public/app/features/dashboard/panel_editor/QueryOptions.tsx

```diff
@@ -71,7 +71,7 @@ export class QueryOptions extends PureComponent<Props, State> {
       tooltipInfo: (
         <>
           The maximum data points the query should return. For graphs this is automatically set to one data point per
-          pixel.
+          pixel. For some data sources this can also be capped in the datasource settings page.
```

Comment:
Not sure if it makes sense to add this at this moment, seemed to me like it would clarify things a bit more.